django/text_extractor/views.py

```diff
@@ -70,10 +71,14 @@ def submit_document(request):
                 )
                 files.insert(0, toc_file)
 
-            current_time = datetime.now().strftime("%Y%m%d_%H%M%S")
-
             for idx, file in enumerate(files):
-                ocr_file, txt_file = create_searchable_pdf(file, merged and idx > 0)
+                bind_contextvars(
```

Comment:
saving the cost in dashboard

Reply:
Good - you can just put this at the top of the submit_document function though, rather than in this loop. It just needs to be set once.